qualtran/bloqs/phase_estimation/qubitization_qpe_test.py

```diff
@@ -70,21 +72,42 @@ def test_qubitization_phase_estimation_of_walk(num_terms: int, use_resource_stat
     state_prep = (
         LPResourceState(precision) if use_resource_state else RectangularWindowState(precision)
     )
-    gh = GateHelper(QubitizationQPE(walk, ctrl_state_prep=state_prep))
-    qpe_reg, selection, target = (gh.quregs['qpe_reg'], gh.quregs['selection'], gh.quregs['target'])
+    qpe_bloq = QubitizationQPE(walk, state_prep)
+
+    def should_decompose(binst):
```

Comment:
can you open an issue and link it here. I assume this isn't ideally how things would work?